Project.toml

```diff
@@ -6,6 +6,7 @@ version = "1.0.0"
 [deps]
 ArgParse = "c7e460c6-2fb9-53a9-8c5b-16f535851c63"
 DelimitedFiles = "8bb1440f-4735-579b-a4ab-409b98df4dab"
+Fortran90Namelists = "8fb689aa-71ff-4044-8071-0cffc910b57d"
```

Comment:
@anchal-physics you should have now access to https://github.com/ProjectTorreyPines/Fortran90Namelists.jl

As we are moving towards opening up our work, I think it would make sense to make a new package, say `Fortran90NamelistsReadWrite.jl` that depends on the original `Fortran90Namelists.jl` here https://github.com/singularitti/Fortran90Namelists.jl and just adds the functionalities of these two commits:
https://github.com/ProjectTorreyPines/Fortran90Namelists.jl/commit/db4fa193d3caba6110131dac031b808a8daab86f
https://github.com/ProjectTorreyPines/Fortran90Namelists.jl/commit/3b5b950c8a960c6ece1bf9338047dc2668d9cc2a

I don't have the bandwidth right now to do this. Would you be willing to create such `Fortran90NamelistsReadWrite.jl` package and make it Apache 2.0 publicly available.

One issue is that `singularitti/Fortran90Namelists.jl` is not a package that is part of the general Julia registry
I see a few options here:
1. we ask @singularitti to register the original package
2. we fork and we register our fork of the original package (I feel this would be lame)
3. we start our own PTP registry for packages that we want to make public (or packages that are not registered in the public julia registry)

I think we must do 3

Reply:
Hi @orso82, I did want to register [Fortran90Namelists.jl](https://github.com/singularitti/Fortran90Namelists.jl). The only problem is that it is a toy now and has not been finished. I only rewrite the lexer in [`f90nml`](https://pypi.org/project/f90nml/). However, its parser is hard to read so I have not finished that part. I will appreciate if you know how to do that!

Reply:
Hi @singularitti !

You can use this tutorial to register your `Fortran90Namelists.jl` package. This would be ideal.
https://julialang.org/contribute/developing_package/#step_5_register_your_package
If the API is not stable you could give it a version 0.x to indicate that.

Could you also please take a look at this commit https://github.com/ProjectTorreyPines/Fortran90Namelists.jl/commit/db4fa193d3caba6110131dac031b808a8daab86f and let me know if you'd be interested incorporating this directly into your repository? This add two functions `readnml` and `writenml` to read/write namelists from/to file. If so, I can open a PR on your side. If not, that's ok, we'll just keep this into a separate repo that has your `Fortran90Namelists.jl` as a dependency.

Thanks!

Reply:
I have make [ProjectTorreyPines/Fortran90Namelists.jl](https://github.com/ProjectTorreyPines/Fortran90Namelists.jl) public now

Reply:
With [ProjectTorreyPines/Fortran90Namelists.jl](https://github.com/ProjectTorreyPines/Fortran90Namelists.jl) public, this issue has been resolved. But I agree with your assessment @orso82, if we do not want to change our package names for Julia registry, we must do option 3 as you listed above.

Reply:
Also just in general, being able to use packages that have not been registered with the general registry.
We already maintain a `GAregistry` for internal packages, but I'd like to have  a separate one for public packages... I'd like this to feel less linked to the GA institution, so something like `FUSEregistry` or `PTPregistry`